### PR TITLE
Story/quad 216

### PR DESCRIPTION
**Pull request by snilapwa**

### CODE REVIEW

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
the layout of the sites page is off:
image

**Comment by jdamerow** [comment]
footer is missing on network visualization pages (network and explore):
image

File: Quadriga/src/main/webapp/resources/public/css/footer.css [hunk 1]
```diff
@@ -25,4 +25,18 @@
...
+
+#footerwrap {
+	background-color: #2f2f2f;
+	color: white;
+	padding-top: 40px;
+	padding-bottom: 60px;
+	text-align: left;
+}
+
+#c {
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/resources/public/css/footer.css]: where is 'c' used? not a good name.

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
Otherwise good. Once the last thing is fixed, it's ready to merge!

**Comment by snilapwa** [comment]
'c' function is for copyright footer, it is used in most of the skeleton pages and is defined in main.css.
I have changed its name to copyrightwrap in footer.css for my changes

**Review by jdamerow — APPROVED** [review]

**Comment by jdamerow** [comment]
@snilapwa for the next branch please make sure to lowercase 'Story': story/QUAD-1 instead of Story/QUAD-1

**Pull request merged**